app/dao/UserDao.scala

```diff
@@ -14,20 +14,25 @@ class UserDao @Inject()(repo: UserRepo)(implicit ec: ExecutionContext)
     with Logging {
 
   override def ensureExists(): Future[Unit] = {
-    for {
-      _ <- super.ensureExists()
-      user <- repo.findByName("admin")
-      if user.isEmpty
```

Comment:
This was throwing on else since Future was failed. 
Reworked into more elegant approach